data/clusters/appint-ex-01/cluster.yml

```diff
@@ -43,15 +43,26 @@ automationToken:
 awsInfrastructureAccess:
 - awsGroup:
     $ref: /aws/app-int-example-01/groups/admin.yml
+  accessLevel: read-only
```

Comment:
since the purpose of this repo is to have examples for how a-i works, it would be great to have a couple of comments on the non-obvious places, e.g. what is the result of awsInfrastructureAccess and what is the result of awsInfrastructureManagementAccounts

Reply:
Fair point. Added some comments now :+1:

Reply:
the code has already been changed so that awsInfrastructureAccess grants access to the users in the mentioned group. it is the awsInfrastructureManagementAccounts section that grants access to the terraform user

Reply:
I see. Adjusted the comments now :+1: